packages/frontend/core/src/components/affine/setting-modal/general-setting/index.tsx

```diff
@@ -32,6 +35,10 @@ export const useGeneralSettingList = (): GeneralSettingList => {
   const hasPaymentFeature = useLiveData(
     serverConfig.features$.map(f => f?.payment)
   );
+  const userFeatureService = useService(UserFeatureService);
```

Comment:
There are many `useService` inside this component, I suggest using `useServices` instead of them, it can be clearer.

eg. https://github.com/toeverything/AFFiNE/blob/35ce4adffeacc76e6acc0effc7b05043b742ecb7/packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/index.tsx#L260-L263

Reply:
Done. Appreciate for your advice!